code/game/gamemodes/changeling/powers/transform.dm

```diff
@@ -7,17 +7,138 @@
 	req_human = 1
 	max_genetic_damage = 3
 
+//changeling clothing items
+/obj/item/clothing/head/changeling

```

Comment:
Fucking, Masks are defined as actual masks, most other slots are /item. looks like it isn't possible (unless you repath mask code to not assume type, like all other slots)